data/map.txt

```diff
@@ -26752,6 +26779,26 @@ planet Starcross
 		threshold 2500
 		fleet "Small Militia" 22
 
+planet "Starting Rubin"
+	attributes kimek "coalition station" shipping wealthy
+	landscape land/space3
+	description `Starting Rubin was far from being the first station constructed around the Kimek homeworld, but it is the oldest one to still be inhabited. It was built as an orbital equivalent to the hunger towers found on the planet it orbits, as a safety measure against the disasters that may affect Ki Patek Ka.`
+	description `Since the residents need few of the food produced, it soonly became the station's primary export, transforming it in one of the largest trading hubs of the region. It has also the largest population of every Coalition station, capable of supporting tens of thousands of Kimek.`
+	spaceport `Despite only having the population of a small city, the arrivals deck of Starting Rubin are as crowded as the largest spaceports. Civilian ships, belonging to all the three Coalition species, can be seen arriving or departing every couple minutes.`
+	spaceport `While there doesn't seem to be any military facility, there are some civilian shops on the station, although they sell nothing but few Kimek items.`
+	shipyard Kimek
+	outfitter "Coalition Basics"
+	"required reputation" 10
+
+planet "Station Cian"
+	attributes kimek "coalition station" shipping
+	landscape land/space6
+	description `Cian Station orbits a large gas giant, which simultaneously has compounds usable in hyperspace fuel, and enough mass to pull an abnormally large numbers of asteroids near it. The local industry centres mostly around the metals harvested here, but part of it is still focused on refining deuterium. Its remote location prevents it from becoming a mayor port, but most traders that travel into this region stop here, even if just to refuel and to watch its impressive facilities.`
+	description `This is also the most recent Coalition station, although it's much older than any in human space.`
+	spaceport `Station Cian seems to acts as base for the military operations in this area, having even one small Heliarch shipyard in a restricted section. Considering the isolationist planet Remote Blue is only one jump away, it's likely the station is also being used as an observational outpost, keeping an eye on the ships that travel that far.`
```

Comment:
`, having even one small Heliarch shipyard in a restricted section.` -> `; it even has a small Heliarch shipyard in a restricted section.`

Reply:
Fixed